### PR TITLE
Add broker ingress liveness probe

### PR DESCRIPTION
**Pull request by yolocs**
Fixes https://github.com/google/knative-gcp/issues/882

**Release Note**

```release-note
Add broker ingress liveness probe
```

**Docs**

### CODE REVIEW

File: pkg/reconciler/brokercell/resources/deployments.go [hunk 1]
```diff
@@ -34,6 +35,20 @@ func MakeIngressDeployment(args IngressArgs) *appsv1.Deployment {
...
 	container.Ports = append(container.Ports, corev1.ContainerPort{Name: "http", ContainerPort: int32(args.Port)})
+	container.LivenessProbe = &corev1.Probe{
+		Handler: corev1.Handler{
+			HTTPGet: &corev1.HTTPGetAction{
+				Path:   "/healthz",
+				Port:   intstr.FromInt(args.Port),
+				Scheme: corev1.URISchemeHTTP,
+			},
+		},
+		FailureThreshold:    3,
```
Comment (grantr) [inline pkg/reconciler/brokercell/resources/deployments.go:46]: Where did these numbers come from?

Comment (yolocs) [reply pkg/reconciler/brokercell/resources/deployments.go:46]: Used the same from Eventing core broker.

Comment (nachocano) [reply pkg/reconciler/brokercell/resources/deployments.go:46]: I recall I had some problems with these numbers on the old implementation... https://github.com/knative/eventing/issues/2060

Comment (yolocs) [reply pkg/reconciler/brokercell/resources/deployments.go:46]: Ack. The fact that Eventing core hasn't fixed the issue might be an indicator that this is not a common problem? We can always adjust these numbers if we find problem.

**Pull request merged**